src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,92 @@
+.product-card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid $border-color;
+  background-color: $background-color;
+
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    object-fit: cover;
+  }
+
+  &__description {
+    margin: 40px 0 16px;
```

Comment:
it is not a good idea - using both top and bottom margins. Choose one and be consistent while adding spacing between elements